lib/doorkeeper/orm/active_record.rb

```diff
@@ -12,6 +12,27 @@ def self.initialize_models!
           end
         end
       end
+
+      def self.initialize_application_owner!
```

Comment:
It's just coincide
- I remember some people want to override it when they implement new ORM adapter
- this is an option, so keep separate between `initialize_models!`